Makefile

```diff
@@ -45,7 +45,6 @@ build-single-arch: pre-build
 
 build-multi-arch: pre-build
 	docker build -f build-image-src/Dockerfile-$(RUNTIME) -t amazon/aws-sam-cli-build-image-$(IS_$(RUNTIME)):x86_64 --platform linux/amd64 --build-arg SAM_CLI_VERSION=$(SAM_CLI_VERSION) --build-arg AWS_CLI_ARCH=x86_64 --build-arg IMAGE_ARCH=x86_64 ./build-image-src
-	docker rmi public.ecr.aws/amazonlinux/amazonlinux:2
```

Comment:
This was required because it re-uses x86 AL2 docker image for arm64 images. Did we confirm it is not required anymore?

Reply:
Yes, before, the `amazonlinux:2` image was being used to being able to use `yum`, that didn't exist on some of the emulation images (Process explained in this comment that we removed: https://github.com/aws/aws-sam-build-images/pull/61/files#diff-5d2845405ef2faf99fcaecfa2260984963917b2fea1e4cd7bd09cff9fc427717)

Now `amazonlinux:2` is not used anymore, so we don't need to delete it.